api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -178,6 +178,7 @@ def from_orm(cls: Any, offer: Offer):  # type: ignore
         offer.expense_domains = get_expense_domains(offer)
         offer.isExpired = offer.hasBookingLimitDatetimesPassed
 
+        offer.stocks = [stocks for stocks in offer.stocks if not stocks.isSoftDeleted]
```

Comment:
(micro-détail / style / question de goût ?) Puisqu'on parle de stock au singulier, je nommerait bien la variable au singulier (et en plus ça facilitera la lecture et évitera une possible confusion entre `offer.stocks` qui est une _liste_ de `Stock`, et `stocks` qui est une _instance_ de `Stock`) : 

```suggestion
        offer.stocks = [stock for stock in offer.stocks if not stock.isSoftDeleted]
```

Reply:
bien vu je suis d'accord